### PR TITLE
User

### PR DESCRIPTION
**Pull request by orangeOAO**
[api]/admin/get-all-users

### CODE REVIEW

File: backend/user/usecase/mail.go
Comment (NOOBDY) [inline backend/user/usecase/mail.go:1]: why hard code value here

**Review by NOOBDY — COMMENTED** [review]
i don't understand

**Review by nonorze-VOXX — APPROVED** [review]
LGTM

File: backend/user/repository/postgresql.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	rows, err := p.db.Query("SELECT * FROM user_data")
+	if err != nil {
+		logrus.Error(err)
+	}
+	l := []swagger.UserData{}
+	for rows.Next() {
+		s := swagger.UserData{}
+		err := rows.Scan(&s.UserId, &s.UserName, &s.Password, &s.UserEmail, &s.Address, &s.Phone, &s.Birthday, &s.Authority, &s.CartId, &s.Status)
+		if err != nil {
+			logrus.Error(err)
```
Comment (nonorze-VOXX) [inline backend/user/repository/postgresql.go:41]: return nil,err

**Pull request merged**